app/controllers/claims_controller.rb

```diff
@@ -27,6 +27,10 @@ def show
       return redirect_to claim_path(current_journey_routing_name, "eligible-itt-subject")
     end
 
+    if params[:slug] == "qualification-details"
+      return redirect_to claim_path(current_journey_routing_name, next_slug) if current_claim.has_no_dqt_data_for_claim?
+    end
```

Comment:
🤔 we should find a solution to remove this logic from the controller. This is here to catch instances where a user has navigated to the page manually. Perhaps the form object should tell us that it can't be instantiated?